setup/setup.go

```diff
@@ -164,6 +169,17 @@ func StartManager(
 		}
 	}
 
+	if operatorOpts.CleanUpOldLogFile {
+		setupLog.V(1).Info("setup log file cleaner", "LogFileMinAge", operatorOpts.LogFileMinAge.String())
+		ticker := time.NewTicker(operatorOpts.LogFileMinAge)
+		go func() {
+			for {
+				<-ticker.C
+				internal.CleanupOldCliLogs(operatorOpts.LogFileMinAge)
```

Comment:
After our chat I think moving this into a go routine with a ticker should be the best option.

Reply:
Agreed!